tools/slackbot/src/release-bot/utils/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { isUndefined } from 'lodash';
+
+/* UTILS */
+
+export function exists<T extends string | Array<any>>(
+  arg: T | undefined,
+): arg is T {
+  return !isUndefined(arg) && arg.length > 0;
+}
+
+export function isValidJSON(str?: string): boolean {
```

Comment:
Is there a way to share this? It looks like you're also using a few lines above.

Reply:
@shaneeza What do you mean share? Like add it to lib/meta?

Reply:
Yeah, or another appropriate place. Cause right now, you're writing this function in both `tools/slackbot` and `tools/meta`

Reply:
👍